packages/dht/test/integration/ReplicateData.test.ts

```diff
@@ -67,6 +61,7 @@ describe('Replicate data from node to node in DHT', () => {
         await Promise.all([
             ...nodes.map(async (node) => await node.stop())
         ])
+        await entryPoint.stop()
```

Comment:
This is not needed, on line 46 entrypoint is pushed among the nodes.

Reply:
Refactored so that `entryPoint` is kept separate from other nodes